src/organisms/cards/ContactCard/index.js

```diff
@@ -0,0 +1,32 @@
+import React, { PropTypes } from 'react';
+import icons from 'assets/stylesheets/utilities/_icons_font.scss';
+import styles from './contact_card.module.scss';
+import classnames from 'classnames';
+
+function ContactCard ( props ) {
+  const {
+    phoneNumber,
+    className
+  } = props;
+
+  return (
+    <ul className={ classnames(className, styles.contact_card) }>
+      <li><a className={ classnames(icons['i-phone']) } href="#">{ phoneNumber }</a></li>
+      <li><a className={ classnames(icons['i-chat']) } href="#">Chat with an Expert</a></li>
```

Comment:
it might be better to pass in a text prop here, just in case we want to use this card to say something else.

on that note, maybe the icons can be dynamic? could pass in the appropriate props and generate icons based on that